src/components/common/MenuAppBar.tsx

```diff
@@ -24,7 +24,8 @@ import { Colours } from '../../styles/Constants';
 
 interface MenuAppBarProps {
   eventId: string;
-  pageTitle: string;
+  pageTitle: string | React.ReactNode;
```

Comment:
added React.ReactNode so I can pass in a React element directly. This just made it easier to add the location icon, without screwing up current uses of the component